about.html

```diff
@@ -0,0 +1,127 @@
+      <span id="openOnClick"><a href="javascript:void(0);" class="icon" onclick="myFunction()">&#9776</a></span>
+      <nav>
+        <ul>
+          <li><a href="./index.html">Home</a></li>
+          <li>Program</li>
+          <li>Join</li>
+          <li>Sponsor</li>
+          <li>News</li>
+        </ul>
+      </nav>
+      <div class="dropdown">
+        <ul id="options" class="dropdown-content">
+          <li><a href="./index.html">Home</a></li>
+          <li>Program</li>
+          <li>Join</li>
+          <li>Sponsor</li>
+          <li>News</li>
+        </ul>
+      </div>
+      <div>
+        <p>CC Campaing</p>
+      </div>
+    </div>
+  </header>
+  <main>
+    <section class="about_headline">
+      <div class="container">
+        <div>
+          <h1>Talent Land Mexico 2023</h1>
+        </div>
+        <div>
+          <p>Talent Land 2023 will be dedicated to the development of talent, so that the participants can meet with
+            experts from different areas who will share their experience and knowledge.
+            This event is a place for creative minds, entrepreneurs and young talents to find new ideas, new people and
+            new technologies to improve their personal and professional lives.
+            Talent Land 2023 is underway and it will be unique.</p>
+        </div>
+        <div>
+          <p>Please contact us per email for any further questions about CC Talent Land Mexico.</p>
+          <p>info@talent-network.org</p>
+        </div>
+      </div>
+    </section>
+
+    <section class="about_talent container">
+      <h2>Jalisco Talent Land </h2>
+      <hr>
+      <p>Since the first Talent Land we managed to bring together creativity and technology, for the next 2023, the
+        machinery has been set in motion to create new emotions, promote the development of talent and promote
+        entrepreneurship among young people.
+        With the same entrepreneurial spirit that has allowed us to carry out unforgettable experiences on a global
+        level, in April 2023 we will discover new ways to change the world with talent.
+        “Talent Land 2023: the future is yours”
+        Get ready!</p>
+      <div>
+        <img src="./assets/logo-talent-land.png" alt="">
+      </div>
+    </section>
+
+    <section class="past-editions">
+      <h2>See the past editions of Talent Land</h2>
+      <p>Take a look at the last two Talent Land Mexico in Expo Guadalajara</p>
+      <div class="past_images">
+        <div class="box"><img class="image" src="./images/TalentLand_2021.jpg" alt="">
+          <div class="middle">
+            <div class="text">Talent Land 2021</div>
+          </div>
+        </div>
+        <div class="box"><img class="image" src="./images/TalentLand_2022.jpg" alt="">
+          <div class="middle">
+            <div class="text">Talent Land 2022</div>
+          </div>
+        </div>
+      </div>
+    </section>
```

Comment:
- [ ] Your design is missing the approriate layout here. The two images are to arranged horizontally on the desktop and vertically on the mobile. But design is opposite of that. Kindly fix that.

### Your Design Desktop
![your-pix-footer](https://user-images.githubusercontent.com/26597432/233480160-1ba14d89-9c9a-480a-abca-a38a3f0385b6.png)

### Your Desgin Mobile
![your-pix-mobile](https://user-images.githubusercontent.com/26597432/233481010-b603e941-e0f8-4f5c-b9ab-d968d6acde63.png)


### Guiding Template
![temp-pix-footer](https://user-images.githubusercontent.com/26597432/233480205-c53a4aad-d197-40c4-963f-316970ca9902.png)

### Template Mobile
![temp-pix-mobile](https://user-images.githubusercontent.com/26597432/233481083-21d36267-8dbe-4f0f-8497-5e9be39a6ae0.png)